admin/words.js

```diff
@@ -19,7 +19,7 @@ systemDictionary = {
     "Cloud instance:": {                             "en": "Cloud instance",                                  "de": "Cloud-Instanz",                                   "ru": "Cloud драйвер",                                   "pt": "Instância de nuvem",                              "nl": "Cloud instantie",                                 "fr": "Instance de cloud",                               "it": "Istanza Cloud",                                   "es": "Instancia de la nube",                            "pl": "Instancja chmury",                                "zh-cn": "云实例"},
     "Device:": {                                     "en": "Device",                                          "de": "Gerät",                                           "ru": "Устройство",                                      "pt": "Dispositivo",                                     "nl": "Apparaat",                                        "fr": "Dispositif",                                      "it": "Dispositivo",                                     "es": "Dispositivo",                                     "pl": "Urządzenie",                                      "zh-cn": "装置"},
     "Drunk:": {                                      "en": "Drunk",                                           "de": "Betrunken",                                       "ru": "Пьяный голос",                                    "pt": "Bêbado",                                          "nl": "Dronken",                                         "fr": "Ivre",                                            "it": "Ubriaco",                                         "es": "Borracho",                                        "pl": "Pijany",                                          "zh-cn": "喝醉了"},
-    "Emotion:": {                                    "en": "Emotion",                                         "de": "Emotion",                                         "ru": "Емоциональный окрас",                             "pt": "Emoção",                                          "nl": "Emotie",                                          "fr": "Émotion",                                         "it": "Emozione",                                        "es": "Emoción",                                         "pl": "Emocja",                                          "zh-cn": "情感"},
```

Comment:
Please also adjust the admin/i18n files because these are needed by json config - changes in word.js are kind of ignored here

Reply:
Does this mean the words.js can be deleted, as there is no other config interface as jsonConfig?

Reply:
When I'm informed correctly yes, but no idea if this makes issues with Weblate? @UncleSamSwiss ?

Reply:
If there is no HTML config, but only JSON: yes, you can delete words.js and please also open an issue at https://github.com/ioBrokerTranslator/requests/ so I can remove the automatic update of words.js.